ui/src/dataLoaders/components/TelegrafEditor.tsx

```diff
@@ -0,0 +1,105 @@
+// Libraries
+import React, {PureComponent, createRef} from 'react'
+import {connect} from 'react-redux'
+
+import {ErrorHandling} from 'src/shared/decorators/errors'
+import {AppState} from 'src/types'
+import {Grid, Columns} from '@influxdata/clockface'
+import TelegrafEditorSidebar from 'src/dataLoaders/components/TelegrafEditorSidebar'
+import TelegrafEditorMonaco from 'src/dataLoaders/components/TelegrafEditorMonaco'
+
+import {
+  TelegrafEditorActivePlugin,
+  TelegrafEditorBasicPlugin,
+  TelegrafEditorBundlePlugin,
+} from 'src/dataLoaders/reducers/telegrafEditor'
+
+import './TelegrafEditor.scss'
```

Comment:
quick question before we lay down a convention ruling. when the initial design was done for the "everything referenced in one file" spec, was there dynamic importing / code splitting being done? seems like a bad idea to have to include / parse / validate / deal with collisions on an inactive section of code. with this way, all of that is removed from the workspace until a feature flag is turned on. also i'd love context on why the giant file instead of having styling following the component, as the feature flag line of reasoning inevitably ends with "why can't we arbitrarily filter all features on a single include?"

Reply:
I don't have any personal opinion on this.. summoning @alexpaxton  :)

Reply:
@drdelambre we used to import stylesheets into their respective components but that caused some excessive duplication and I think there was something else that made webpack unhappy. We ended up backtracking and having the stylesheets all roll up into a single stylesheet until we had time to create a better solution.

I would love if stylesheets could be imported into specific components because it feels more discreet and organized. However each stylesheet would need to be able to access global scss variables and mixins. Ideally this is possible without duplication issues

Reply:
i assumed that this issue was fixed with the switch from `ExtractTextPlugin` to `MiniCssExtractPlugin`, but I will double check before proceeding. thanks for the background!